src/features/Managers/Popups/SelectRespondentsPopup/SelectRespondents.const.ts

```diff
@@ -0,0 +1,7 @@
+export const respondents = [
```

Comment:
If it's mocked data, maybe it's better to name it accordingly (for ex. "mockedRespondents")?